### PR TITLE
task 5 done

### PR DESCRIPTION
**Pull request by NikitaSelanko**
Done  separate  page for each category

### CODE REVIEW

File: src/components/ReviewCard.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 export default function ReviewCard({title, owner, votes,}) {
     return(
         <article className="ReviewBox">
-            <h3 className="ReviewOwner">{owner}</h3>
             <div className="ReviewBoxDetails">
-            <h4 className="ReviewTitle">{title}</h4>
             <p className="ReviewVotes">{votes}</p>
+            <h3 className="ReviewOwner">{owner}</h3>
+            <h4 className="ReviewTitle">Title:{title}</h4>
```
Comment (CatHoang) [inline src/components/ReviewCard.jsx:7]: I would expect this to be the main element on the page but is currently the smallest element, normally the title is at the top and is in large font.

File: src/components/ReviewCard.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 export default function ReviewCard({title, owner, votes,}) {
     return(
         <article className="ReviewBox">
-            <h3 className="ReviewOwner">{owner}</h3>
             <div className="ReviewBoxDetails">
-            <h4 className="ReviewTitle">{title}</h4>
             <p className="ReviewVotes">{votes}</p>
```
Comment (CatHoang) [inline src/components/ReviewCard.jsx:5]: I would maybe name this and currently it just displays a number which is quite confusing as the user has no idea what this number is referring to.

**Review by CatHoang — COMMENTED** [review]
Hi Nikita, I have left some feedback for you to think about - I would also look at the responsive of your app. At the moment when it is on desktop, it is all squished to the left hand side. Feel free to merge

File: src/components/ReviewCard.jsx (hunk 1 above)
Comment (NikitaSelanko) [reply src/components/ReviewCard.jsx:7]: ye i was having issue with flex, it was jumping around and i was playing with them until they got to perfect position

File: src/components/ReviewCard.jsx (hunk 2 above)
Comment (NikitaSelanko) [reply src/components/ReviewCard.jsx:5]: yes later on i want to add icon, just currently i've forcing myself to focus on code not on style))

**Pull request merged**